src/bvals/comms/bnd_info.hpp

```diff
@@ -69,9 +69,9 @@ struct BndInfo {
   Coordinates_t coords, coarse_coords; // coords
 
   buf_pool_t<Real>::weak_t buf;         // comm buffer from pool
-  ParArray6D<Real, VariableState> var;  // data variable used for comms
-  ParArray6D<Real, VariableState> fine; // fine data variable for prolongation/restriction
-  ParArray6D<Real, VariableState>
+  ParArrayND<Real, VariableState> var;  // data variable used for comms
+  ParArrayND<Real, VariableState> fine; // fine data variable for prolongation/restriction
+  ParArrayND<Real, VariableState>
```

Comment:
These are now seven dimensional arrays under the hood since we need another index for potentially denoting the face/edge we are asking for. We could go to `ParArray7D` instead everywhere, I am not invested in either choice.

Reply:
I'll leave this open for now in case anyone else has an opinion.